docker/startup.sh

```diff
@@ -1,13 +1,51 @@
 #!/bin/sh
 
-if [ ! -d /app/.github ] || [ ! -d /home/user/.cache ]; then
+if [ ! -d /app/.github ] || [ ! -d ~/.cache ]; then
     echo " "
     echo "ERROR: Docker container was not started properly."
     echo "       From the root of this repo, run the following command."
     echo "       You may add any command to perform in the container at the end of this command."
     echo " "
-    echo '  docker run --rm -it -v "$PWD:/app/" -v "$PWD/docker/.cache:/home/user/.cache" maplibre-native-image'
+    echo "  docker run --rm -it -v \"$PWD:/app/\" -v \"$PWD/docker/.cache:/home/$USERNAME/.cache\" maplibre-native-image"
```

Comment:
this is not correct - double quotes would convert `$PWD` to the value **inside** the docker container, whereas the user should just copy/paste this string so that `$PWD` is resolved using the host.

```suggestion
    echo '  docker run --rm -it -v "$PWD:/app/" -v "$PWD/docker/.cache:/home/'"$USERNAME"'/.cache" maplibre-native-image'
```